.ci_support/linux_aarch64_.yaml

```diff
@@ -0,0 +1,20 @@
+BUILD:
+- aarch64-conda_cos7-linux-gnu
+cdt_arch:
+- aarch64
+cdt_name:
+- cos7
+channel_sources:
+- conda-forge,c4aarch64,defaults
+channel_targets:
+- conda-forge main
+ctng_binutils:
+- 2.29.1
```

Comment:
@jjhelmus, what was the reason to choose 2.29.1 while it is 2.31.1 on all other platforms?

Reply:
binutils >=2.30 require glibc 2.24 which is newer than the CentOS 7 target of 2.17, see:
https://github.com/crosstool-ng/crosstool-ng/commit/36bb675a71cc66a2abd69bc82d613f2153d3791f